syntax = "proto3";
package proto;

message InstructionRef
{
  bytes block_id = 1;
  uint64 offset = 2;
}
